src/Http/ApiProblem/ApiProblem.php

```diff
@@ -407,4 +408,13 @@ public static function labelNotAllowed(string $labelName): self
         $e->setExtraProperties(['label' => $labelName]);
         return $e;
     }
+
+    public static function useAttendanceMode(string $detail): self
+    {
+        return self::pathParameterInvalid(
```

Comment:
If/when we refactor the "update location" endpoint to use a JSON body (so an integrator can also use the complete URL of a location, instead of just the UUID), this type will be incorrect and I think it will be easy to glance over because it's now hidden inside `useAttendanceMode` 🤔

Reply:
I inlined the usage of `pathParameterInvalid` again and made a private method to create the message.